src/unstract/sdk/adapters/x2text/llm_whisperer/src/llm_whisperer.py

```diff
@@ -360,22 +371,36 @@ def _write_output_to_file(self, output_json: dict, output_file_path: Path) -> No
         try:
             text_output = output_json.get("text", "")
             logger.info(f"Writing output to {output_file_path}")
-            output_file_path.write_text(text_output, encoding="utf-8")
+            fs.write(
+                path=output_file_path.name,
+                mode="w",
+                encoding="utf-8",
+                data=text_output,
+            )
+            # output_file_path.write_text(text_output, encoding="utf-8")
```

Comment:
remove this commented lines if not required

Reply:
Done